src/test/java/seedu/intrack/logic/parser/AddCommandParserTest.java

```diff
@@ -97,63 +97,63 @@ public void parse_compulsoryFieldMissing_failure() {
-                + ADDRESS_DESC_MSFT, expectedMessage);
+                + WEBSITE_DESC_MSFT, expectedMessage);
 
         // missing name prefix
         assertParseFailure(parser, NAME_DESC_MSFT + VALID_POSITION_MSFT + PHONE_DESC_MSFT + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT, expectedMessage);
+                + WEBSITE_DESC_MSFT, expectedMessage);
 
         // missing phone prefix
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + VALID_PHONE_MSFT + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT, expectedMessage);
+                + WEBSITE_DESC_MSFT, expectedMessage);
 
         // missing email prefix
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + PHONE_DESC_MSFT + VALID_EMAIL_MSFT
-                + ADDRESS_DESC_MSFT, expectedMessage);
+                + WEBSITE_DESC_MSFT, expectedMessage);
 
-        // missing address prefix
+        // missing website prefix
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + PHONE_DESC_MSFT + EMAIL_DESC_MSFT
-                + VALID_ADDRESS_MSFT, expectedMessage);
+                + VALID_WEBSITE_MSFT, expectedMessage);
 
         // all prefixes missing
         assertParseFailure(parser, VALID_NAME_MSFT + VALID_POSITION_MSFT + VALID_PHONE_MSFT + VALID_EMAIL_MSFT
-                + VALID_ADDRESS_MSFT, expectedMessage);
+                + VALID_WEBSITE_MSFT, expectedMessage);
     }
 
     @Test
     public void parse_invalidValue_failure() {
         // invalid name
         assertParseFailure(parser, INVALID_NAME_DESC + POSITION_DESC_MSFT + PHONE_DESC_MSFT + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Name.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Name.MESSAGE_CONSTRAINTS);
 
         // invalid position
         assertParseFailure(parser, NAME_DESC_MSFT + INVALID_POSITION_DESC + PHONE_DESC_MSFT + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Position.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Position.MESSAGE_CONSTRAINTS);
 
         // invalid phone
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + INVALID_PHONE_DESC + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Phone.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Phone.MESSAGE_CONSTRAINTS);
 
         // invalid email
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + PHONE_DESC_MSFT + INVALID_EMAIL_DESC
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Email.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Email.MESSAGE_CONSTRAINTS);
 
         // invalid address
```

Comment:
```suggestion
        // invalid website
```
I think this documentation change was missed out.

Reply:
Thank you for pointing that out, I have fixed the documentation.